src/igvfd/audit/file_set.py

```diff
@@ -176,3 +187,61 @@ def audit_inconsistent_seqspec(value, system):
                     f'but belong to different sequencing sets.'
                 )
                 yield AuditFailure('inconsistent sequence specifications', f'{detail} {description}', level='ERROR')
+
+
+@audit_checker('FileSet', frame='object')
```

Comment:
It will only affect file sets with the property small_scale_loci_list (i.e. Construct library sets & prediction sets), I don't see the harm in leaving it generally applied to all file sets in case these properties are added elsewhere. What do you think?

Reply:
the "harm" is calling the function on ALL file sets vs. only subset of file sets - making it less computationally expensive

Reply:
Ok I changed it to only affect CLS and prediction sets